terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars

```diff
@@ -2,5 +2,6 @@ environment = "live"
 aws_profile = "live-eu-west-2"
 
 # service configs
-use_set_environment_files = true
-desired_task_count = 2 # use multi instance in stagingenvironment = "live"
\ No newline at end of file
+log_level = "info"
```

Comment:
```suggestion

```
Since the log_level is defined in the [S3](https://github.com/companieshouse/ecs-service-configs-dev/blob/main/development-eu-west-2/cidev/confirmation-statement-web.env) .env file, the log_level from these profile/vars can be removed.